packages/charts/src/charts/bar/components/bars/bars.component.tsx

```diff
@@ -30,6 +30,7 @@ export type Props = {
   groupMode: GroupMode;
   showValues: boolean;
   valuesAutocolor: boolean;
+  activeKey?: string;
```

Comment:
What do You think about change type definition to `string | null`. Then in `useEffect` also change this strange `typeof` condition:
```
 if (isActive === null) {
     return;
 }
```

Reply:
We could also set `activeKey` default as a `null`